docs/092-ht_change_voice.md

```diff
@@ -1,3 +1,54 @@
 # How To Change Your Assistant's Voice
 
-**WIP**
+## Using TTS server plugin
+
+TTS server plugin is using services hosted by OVOS community:
```

Comment:
While this is true by default, anybody can host their own server also.
There is an excellent blog post about this here
https://blog.graywind.org/posts/piper-tts-server-script/